dojo/finding/views.py

```diff
@@ -2979,9 +2979,9 @@ def finding_bulk_update_all(request, pid=None):
 
                 error_counts = defaultdict(lambda: 0)
                 success_count = 0
-                finding_groups = set(
-                    [find.finding_group for find in finds if find.has_finding_group]
-                )
+                finding_groups = {
+                    find.finding_group for find in finds if find.has_finding_group
+                }  # noqa: C403
```

Comment:
I think I may have missed this one in my initial comment yesterday

```suggestion
                finding_groups = set(
                    find.finding_group for find in finds if find.has_finding_group
                )  # noqa: C403
```

Reply:
I do not know how but you are right. I missed this one. Thank you.